sample_size/scripts/sample_size_run.py

```diff
@@ -15,19 +15,32 @@ def main() -> None:
     from sample_size.sample_size_calculator import SampleSizeCalculator
     from sample_size.scripts.input_utils import get_alpha
     from sample_size.scripts.input_utils import get_metric_metadata
+    from sample_size.scripts.input_utils import get_mde
+    from sample_size.scripts.input_utils import get_variants
+    from sample_size.scripts.input_utils import register_another_metric
 
     try:
         # Get alpha for power analysis
         alpha = get_alpha()
-        calculator = SampleSizeCalculator(alpha)
+        variants = get_variants()
+        calculator = SampleSizeCalculator(alpha, variants)
 
-        # register metric
-        metric_type, metric_metadata = get_metric_metadata()
-        calculator.register_metric(metric_type, metric_metadata)
+        register = True
+        while register:
+            metric_type, metric_metadata = get_metric_metadata()
+            for v in range(int(variants) - 1):
+                print(f'Treatment {v + 1}')
```

Comment:
why are we printing out the treatment number?

Reply:
Oh it's supposed to be printed only when variants>2. For each treatment we are only asking for MDE so users don't need to enter other metadata again. For example if you have 3 treatments, the calculator will ask the following
```
Metric type
baseline
MDE

Treatment 2
MDE

Treatment 3
MDE

Are you going to register another metric? (Y/N)
```

To fix this I've changed the code to 
```
  for v in range(int(variants) - 1):
      if v > 0:
          print(f'Treatment {v + 1}')
          metric_metadata["mde"] = get_mde(metric_type)
```

Reply:
hmm...given a metric,  asking for different MDEs per variant may confuse people? It is hard for them to determine one MDE already. Can we just require one MDE per metric? Thoughts? @pchiang-godaddy @vjohnson1-godaddy

Reply:
I agree. From a different perspective, if an experimenter is having trouble determining a MDE, perhaps they could enter a range? From our end, it doesn't add much complexity to support a uniform prior on the parameters we're using to generate the p-values. Might be a good V2 / V3 discussion

Reply:
Do the users usually apply same MDE for all treatment variants? If that's the case then we will use one MDE for treatments. Same for the MDE range - if determining a specific MDE is often a blocker for the users, I'm totally fine with adding this feature.